lib/makara/connection_wrapper.rb

```diff
@@ -97,12 +103,24 @@ def execute(*args)
           args[0] = replace
         end
       end
+      run_queue if ENV["MAKARA_LAZY_CONNECT"] == "true"
```

Comment:
I think instead of queuing, you could run initialize here if it has not been initialized already.

Reply:
I think we will only have a `ConnectionWrapper` object if the pool is already initialized. Before initialization we only have a pool object without any wrapped connections.